src/xo-app/host/tab-patches.js

```diff
@@ -151,13 +171,15 @@ export default class HostPatches extends Component {
             {isEmpty(missingPatches)
               ? <TabButton
                 disabled
-                handler={installAllPatches}
+                handler={this._installAllPatchesWarning}
```

Comment:
Yep that makes sense, on the other hand, `TabButton` requires such a prop, maybe use the condition directly in the `disabled` prop.